distribution/ddf-common/src/main/resources/bin/ddf

```diff
@@ -32,7 +32,7 @@ SOLR_EXEC=${HOME_DIR}/solr/bin/solr
 get_property() {
 
     value=""
-    value=`grep ^$1= "$PROPERTIES_FILE" | cut -d = -f2`
+    value=`grep ^$1 "$PROPERTIES_FILE" | cut -d = -f2`
```

Comment:
Why?

Reply:
allow script to support properties with spaces around `=`. For example `solr.port = 8994`. Configuration in Dev/I&T has spaces around properties.